.gitignore

```diff
@@ -2,4 +2,5 @@
 .DS_Store
 .env
 .clone
```

Comment:
should `.clone` not be removed then?

Reply:
keeping this around for backward compatibility. (i.e., want to avoid the situation where people with a left-over `.clone` directory accidentally add it to Git.) we can remove it at a later point when we're no longer concerned that people may be using old checkouts